lightfm/lightfm.py

```diff
@@ -418,7 +418,7 @@ def _check_input_finite(self, data):
             raise ValueError('Not all input values are finite. '
                              'Check the input for NaNs and infinite values.')
 
-    def fit(self, interactions,
+    def fit(self, interactions, scope=None, match_indices=None,
```

Comment:
Can you please update docstring? No `scope` and `match_indices` - I guess those args replaced `menu`?